semantic-python/test/fixtures/1-05-exec-statement-fails.py

```diff
@@ -0,0 +1,2 @@
+# CHECK-FAILS
+exec "'We will never support exec statements.'"
```

Comment:
Turns out this was failing not because it was an `eval`, but because `eval` is not special syntax. To trigger this syntax you have to use `exec` as a statement in the manner of Python 2.7.

Reply:
Wow!